apps/route_patterns/lib/repo.ex

```diff
@@ -10,6 +10,7 @@ defmodule RoutePatterns.Repo do
       nil -> [route: route_id]
       direction_id -> [route: route_id, direction_id: direction_id]
     end
+    |> Keyword.put(:sort, "sort_order")
```

Comment:
Is this what you were referring to earlier when you said the default sort order seemed to be by typicality?

Reply:
yes